5_databases/week1/Homework.sql

```diff
@@ -0,0 +1,57 @@
+select * FROM task;
+SELECT * from status;
+select * FROM user;
+
+1--Find out how many tasks are in the task table
+
+SELECT COUNT(id) from `task`;
+
+2--Find out how many tasks in the task table do not have a valid due date
+
+SELECT COUNT(id) FROM `task` WHERE due_date IS  NULL;
+
+3--Find all the tasks that are marked as done
+
+SELECT task.id,
+       status.name
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name = 'Done';
+
+4--Find all the tasks that are not marked as done
+
+SELECT task.id, status.name
+FROM task
+JOIN status ON task.status_id = status.id
+WHERE status.name != 'Done';
+
+5--Get all the tasks, sorted with the most recently created first
+
+SELECT * FROM `task` ORDER BY created DESC;
+
+6--Get the single most recently created task
+
+SELECT * FROM `task` ORDER BY created DESC LIMIT 1;
+
+7--Get the title and due date of all tasks where the title or description contains database
+
+SELECT title, due_date FROM task WHERE title LIKE '%database%' OR description LIKE '%database%';
+
+8--Get the title and status (as text) of all tasks
+
+SELECT title, status_id FROM task;
```

Comment:
Great work on the `SELECT`s so far! 🎉 For this particular one, the status should be shown as text which I assume means its name. 😄 How would you do that?